webui/react/src/e2e/fixtures/user.fixture.ts

```diff
@@ -51,16 +55,17 @@ export class UserFixture {
     username = safeName('test-user'),
     displayName,
     isAdmin,
+    password = 'testPassword1',
```

Comment:
Can we pull this from an env variable in circle CI? The playwright context should probably be the context for it. We should avoid hardcoding passwords in these tests so that they are environment agnostic.

Reply:
I don't like seeing a default here. it's not a required field, is it?

Reply:
if it's required upon create, I'd suggest a new `EditArgs` interface